internal/pkg/machine/services/machined.go

```diff
@@ -684,6 +688,20 @@ func (c *machineService) Containers(context.Context, *machine.ContainersRequest)
 	return &machine.ContainersResponse{}, nil
 }
 
+// MetaWrite implements machine.MachineServiceServer.
```

Comment:
we could probably make it work as the real meta write and read if you add the resources to the `c.state`